docs/files/backends.rst

```diff
@@ -0,0 +1,55 @@
+..
+    This file is part of Invenio.
+    Copyright (C) 2019 CERN.
+
+    Invenio is free software; you can redistribute it and/or modify it
+    under the terms of the MIT License; see LICENSE file for more details.
+
+.. _backends:
+
+
+Configure your Storage Backend
```

Comment:
It's not clear for me if this section was meant to be a "how-to guide" or a "reference guide" (I am referring to the classification in the article https://www.divio.com/blog/documentation/ ). We can discuss about it :)